flow/flow.go

```diff
@@ -475,6 +462,30 @@ func (d *Flow) updateFromLogs(containerName string) error {
 	return nil
 }
 
+// access nodes (and potentially others) use chain_id to log the network
+var networkLogKeyCandidates = []string{"chain", "chain_id"}
+
+// updateNetworkFromJSONLog update the network by looking for a known value
+// in a list of keys. Network is updated based on the first key with a value
+// containing with a known network.
+func (d *Flow) updateNetworkFromJSONLog(m map[string]interface{}) {
+	for _, key := range networkLogKeyCandidates {
+		if chain, ok := m[key]; ok && d.network == "" {
```

Comment:
I'd suggest extracting the conditional `d.network == ""` to the beginning of a function for an early return.

Reply:
Addressed in 44febfa3a5